engine/runtime/src/main/java/org/enso/interpreter/runtime/data/atom/Atom.java

```diff
@@ -131,12 +131,19 @@ private String toString(int depth) {
 
   @ExportMessage
   boolean hasMembers() {
-    return true;
+    if (constructor.getType().isProjectPrivate()) {
+      return false;
+    } else {
+      return true;
+    }
```

Comment:
```suggestion
    return !constructor.getType().isProjectPrivate());
```
I think we can agree this is more readable?